obspy/core/tests/test_response.py

```diff
@@ -298,6 +298,20 @@ def test_response_with_no_units_in_stage_1(self):
             out, [0 + 9869.2911771081963j, 0 + 19738.582354216393j,
                   0 + 39477.164708432785j])
 
+    def construct_response_from_paz_values(self):
+        poles = [1+2j, 1-2j, 2+2j, 2-2j]
+        zeros = [0, 0, 5]
+        resp = Response.from_paz(zeros, poles)
```

Comment:
Please add a bit more to this test, e.g. make some assertions about the response object and maybe also calculate the response and compare with a response from some known-good source.